infer/src/base/CommandLineOption.ml

```diff
@@ -990,6 +990,7 @@ let parse ?config_file ~usage action initial_command =
          to prevent this from happening *)
       let file = Filename.temp_file "args_" "" in
       Out_channel.with_file file ~f:(fun oc -> Out_channel.output_lines oc argv_to_export) ;
+      Pervasives.at_exit (fun () -> try Unix.unlink file with _ -> ()) ;
```

Comment:
Please use `Epilogues.register` so that this also works when infer is interrupted by a signal (unless this creates a dependency cycle?).